ntbs-integration-tests/HomePage/HomePageTests.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ntbs_integration_tests.Helpers;
+using ntbs_service;
+using ntbs_service.Models.Validations;
+using Xunit;
+
+namespace ntbs_integration_tests.HomePage
+{
+    public class HomePageTests : TestRunnerBase
+    {
+        public HomePageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory) { }
+
+        public const string PageRoute = "/Index";
+        public const string DismissPageRoute = "/Alerts/1/Dismiss";
+
+
+        [Fact]
+        public async Task DismissAlert_CorrectlyDismissesAlertAndReturnsHomePage()
+        {
+            // Arrange
+            var initialPage = await Client.GetAsync(PageRoute);
+            var pageContent = await GetDocumentAsync(initialPage);
+            var formData = new Dictionary<string, string>
+            {
+                // ["AlertId"] = Utilities.ALERT_ID.ToString()
+            };
+
+            // Act
+            Assert.NotNull(pageContent.QuerySelector("tr[id='alert-1']"));

```

Comment:
Is that just making sure the initial page makes sense? If s, should it be part of th arrange block?
I would suggest we don't require it to be a `tr` either, as that ties it to structure quite a lot - the selector can be just `"#alert-id"`

Reply:
Yep it is re making sure initial page is okay - will make suggested changes :)